sdk/tests/conformance/extensions/webgl-draw-buffers-feedback-loop.html

```diff
@@ -129,7 +129,7 @@
 
     gl.bindFramebuffer(gl.FRAMEBUFFER, fbo);
     gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, tex0, 0);
-    gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT1, gl.TEXTURE_2D, tex1, 0);
+    gl.framebufferTexture2D(gl.FRAMEBUFFER, ext.COLOR_ATTACHMENT1_WEBGL, gl.TEXTURE_2D, tex1, 0);
```

Comment:
Since you use `ext` down here now, you should move `ext` into the global scope. Move the declaration (but not the definition) up next to `var fbo`.